packages/hotpotqa/src/aviary/envs/hotpotqa/env.py

```diff
@@ -186,6 +197,10 @@ def __init__(
         self.incorrect_reward = incorrect_reward
         self.tool_failure_reward = tool_failure_reward
         self.proxy = proxy
+        self.evaluation_mode = evaluation_mode
+
+        if evaluation_mode == EvalAnswerMode.LLM_SCORE:
+            raise ValueError(f'Evaluation mode "{evaluation_mode}" is not supported.')
```

Comment:
Can we either say:
- Why we don't support it
- "not yet supported"

Reply:
Went with the former